src/components/graph/menus/network-modification-node-editor.jsx

```diff
@@ -797,14 +805,16 @@ const NetworkModificationNodeEditor = () => {
         );
 
         setSaveInProgress(true);
-        snackInfo({
-            headerId: 'infoCreateModificationsMsg',
-            headerValues: {
-                nbModifications: selectedItems.length,
-                studyDirectory: studyDirName,
-            },
-        });
-        createModifications(studyDirectoryUuid, modificationPropsList)
+        createModifications(studyParentDirectoryUuid, modificationPropsList)
+            .then(() => {
+                snackInfo({
```

Comment:
If moved in then, maybe we should change the message `infoCreateModificationsMsg` to be :
```diff
-"infoCreateModificationsMsg": "Creation of {nbModifications} network modifications in {studyDirectory}"
+"infoCreateModificationsMsg": "{nbModifications} network modifications successfully created in {studyDirectory}"
```
fr:
```diff
-"infoCreateModificationsMsg": "Création de {nbModifications} modifications dans {studyDirectory}",
+"infoCreateModificationsMsg": "{nbModifications} modifications dans {studyDirectory} créée(s) avec succès",
```